isis/src/qisis/apps/ipce/IpceMainWindow.h

```diff
@@ -176,6 +176,13 @@ namespace Isis {
    *                           project was in fullscreen or not when saved. If not, we call showNormal()
    *                           to restore the poject's window size. This also fixes the warning/history tabs
    *                           being misplaced when opening a project. Fixes #5175.
+   *   @history 2018-07-12 Kaitlyn Lee - Removed code that makes the window fullscreen in memory, since this
+   *                           was causing a project's window size to not be restored when opening from the
+   *                           command line. Decreased the size and changed the color of the splitter.
+   *                           In writeGlobalSettings(), check to see if the geometry value does not exist
+   *                           in the config file. This allows the geometry to be saved if the config file
```

Comment:
A number of your lines appear to be going JUST over the limit. Not a huge deal, just makes it a little harder to follow on Git.